### DISCUSSION TITLE
2005/03/16/%E6%88%91%E6%96%B0%E4%B9%B0%E7%9A%84%E6%89%8B%E8%A1%A8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#8] [Announcements]
# 2005/03/16/%E6%88%91%E6%96%B0%E4%B9%B0%E7%9A%84%E6%89%8B%E8%A1%A8

刚开始水平很差，两星期后就滑得很好了。

https://blog.qizhen.xyz/2005/03/16/%E6%88%91%E6%96%B0%E4%B9%B0%E7%9A%84%E6%89%8B%E8%A1%A8

**Comment by ruanqizhen** [comment]
之前博客下的留言：
```
Unknown
2005年03月19日 21:34 
呵呵 是卡西欧的登山吗


hp
2005年03月18日 10:29 
Nice toy. Suitable to wear in sport


Unknown
2005年03月16日 21:10 
臭美！
```
